app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version = "1.0" encoding = "utf-8"?>
+<LinearLayout xmlns:android = "http://schemas.android.com/apk/res/android"
+    xmlns:tools = "http://schemas.android.com/tools"
+    android:layout_width = "match_parent"
+    android:layout_height = "match_parent"
+    android:layout_margin = "16dp"
+    android:orientation = "vertical"
+    tools:context = ".MainActivity"
+    >
+    <TextView
+        android:id="@+id/backgroundtext"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="90dp"
+        android:text="Currently there is a lot of food wastage in the world be it by the restaurant or by the individuals. And people tend to throw away the excess or left over food items.
+The main aim of this application is to not waste food and reach food for the needy"
```

Comment:
Nitpick, but "leftover" is one word.

Maybe create a ticket to improve this copy and brush up the designs, but put it at the bottom of the backlog?

Reply:
yeah sure professor